cmd/fs-v1-helpers.go

```diff
@@ -364,6 +364,26 @@ func fsRenameFile(ctx context.Context, sourcePath, destPath string) error {
 	return nil
 }
 
+// Renames source path to destination path, creates all the
+// missing parents if they don't exist.
+func panfsRenameFile(ctx context.Context, sourcePath, destPath string) error {
```

Comment:
I do link "File" in the name of function as it's clearly declares that it will not work with directories. Can we update names for other functions to emphasize this fact as well?

Reply:
Done